#! /bin/sh /usr/share/dpatch/dpatch-run
## 10license.dpatch by  <lamont@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: BSD License allows dropping of clause 3 from the 4-term license.

@DPATCH@
diff -urNad util-linux/login-utils/wall.1 /tmp/dpep.ZDTG6n/util-linux/login-utils/wall.1
--- util-linux/login-utils/wall.1	2002-01-06 07:05:23.000000000 -0700
+++ /tmp/dpep.ZDTG6n/util-linux/login-utils/wall.1	2004-12-15 07:48:33.739391993 -0700
@@ -9,11 +9,7 @@
 .\" 2. Redistributions in binary form must reproduce the above copyright
 .\"    notice, this list of conditions and the following disclaimer in the
 .\"    documentation and/or other materials provided with the distribution.
-.\" 3. All advertising materials mentioning features or use of this software
-.\"    must display the following acknowledgement:
-.\"	This product includes software developed by the University of
-.\"	California, Berkeley and its contributors.
-.\" 4. Neither the name of the University nor the names of its contributors
+.\" 3. Neither the name of the University nor the names of its contributors
 .\"    may be used to endorse or promote products derived from this software
 .\"    without specific prior written permission.
 .\"
diff -urNad util-linux/login-utils/wall.c /tmp/dpep.ZDTG6n/util-linux/login-utils/wall.c
--- util-linux/login-utils/wall.c	2002-03-08 16:00:19.000000000 -0700
+++ /tmp/dpep.ZDTG6n/util-linux/login-utils/wall.c	2004-12-15 07:48:33.748390902 -0700
@@ -10,11 +10,7 @@
  * 2. Redistributions in binary form must reproduce the above copyright
  *    notice, this list of conditions and the following disclaimer in the
  *    documentation and/or other materials provided with the distribution.
- * 3. All advertising materials mentioning features or use of this software
- *    must display the following acknowledgement:
- *	This product includes software developed by the University of
- *	California, Berkeley and its contributors.
- * 4. Neither the name of the University nor the names of its contributors
+ * 3. Neither the name of the University nor the names of its contributors
  *    may be used to endorse or promote products derived from this software
  *    without specific prior written permission.
  *
